src/test/java/module18/endpoints/PetStorePetEndpoint.java

```diff
@@ -21,6 +23,7 @@
 public class PetStorePetEndpoint {
```

Comment:
craete private constructor as this class is Singleton. You should have only 1 instance at any time which obtained by `PetStorePetEndpoint.getInstance()`